addons/web/views/webclient_templates.xml

```diff
@@ -598,6 +598,21 @@
         </t>
     </template>
 
+    <template id="web.app_auth" name="Accept app">
+        <t t-call="web.login_layout">
+            <form class="oe_login_form" role="form" method="post" action="/web/oauth">
+                <input type="hidden" name="csrf_token" t-att-value="request.csrf_token()"/>
+                <input type="hidden" name="redirect" t-att-value="redirect"/>
+                <input type="hidden" name="scope" t-att-value="scope"/>
+
+                <t t-esc="scope" /> wants to access Odoo.
```

Comment:
I called the parameter `friendlyname` instead of `label`, but `label` was probably a better idea. So now the Login Page is called with 3 parameters. The `redirect`, `scope`, `friendlyname`, and `info` which are suggestions of additional info to add to the name of the key. Then, on the page with Allow/Deny we can add information to the name of the key. The example here shows the browser, the platform, and the date, that we all can get from javascript rather reliably (getting the browser name is actually tricky (https://stackoverflow.com/a/9851769/1510212) :
![image](https://user-images.githubusercontent.com/372102/89896755-47ad8c00-dbde-11ea-9d60-4fc19e716883.png)

Then the friendly name and additional info are concatenated for the description (name) of the key:
![image](https://user-images.githubusercontent.com/372102/89896827-60b63d00-dbde-11ea-8e0a-038a4525d1fb.png)